internal/tracker/source/tekton/tekton.go

```diff
@@ -348,23 +351,40 @@ func PreparePackage(i *PreparePackageInput) (*hub.Package, error) {
 	var name, version, description, tektonKind string
 	var annotations map[string]string
 	var tasks []map[string]interface{}
+	var steps []v1beta1.Step
+	var containerImages []*hub.ContainerImage
```

Comment:
Can we move the `containerImages` declaration to the top of the block where the images are prepared below? (L428)

Reply:
Sure, updated~